.github/workflows/test-vrl-web-playground.yml

```diff
@@ -0,0 +1,94 @@
+name: Test VRL Web Playground
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  changes:
+      runs-on: ubuntu-20.04
+      # Set job outputs to values from filter step
+      outputs:
+        source: ${{ steps.filter.outputs.source }}
+        dependencies: ${{ steps.filter.outputs.dependencies }}
+        internal_events: ${{ steps.filter.outputs.internal_events }}
+        cue: ${{ steps.filter.outputs.cue }}
+        markdown: ${{ steps.filter.outputs.markdown }}
+      steps:
+      - uses: actions/checkout@v3
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            source:
+              - ".github/workflows/test.yml"
+              - ".cargo/**"
+              - "benches/**"
+              - "lib/**"
+              - "proto/**"
+              - "scripts/**"
+              - "skaffold/**"
+              - "src/**"
+              - "tests/**"
+              - "build.rs"
+              - "Cargo.lock"
+              - "Cargo.toml"
+              - "Makefile"
+              - "rust-toolchain.toml"
+            deny:
+              - 'deny.toml'
+            dependencies:
+              - ".cargo/**"
+              - 'Cargo.toml'
+              - 'Cargo.lock'
+              - 'rust-toolchain.toml'
+              - '.github/workflows/pr.yml'
+              - 'Makefile'
+              - 'scripts/cross/**'
+            cue:
+              - 'website/cue/**'
+            markdown:
+              - '**/**.md'
+            internal_events:
+              - 'src/internal_events/**'
+            docker:
+              - 'distribution/docker/**'
+
+  test-vrl-web-playground:
+    name: VRL Web Playground - Linux
+    continue-on-error: true
+    runs-on: [linux, test-runner]
+    needs: changes
+    if: ${{ needs.changes.outputs.source == 'true' || needs.changes.outputs.cue == 'true' }}
+    steps:
+      - uses: actions/checkout@v3
+      - run: sudo -E bash scripts/environment/bootstrap-ubuntu-20.04.sh
+      - run: bash scripts/environment/prepare.sh
+      - run: cd lib/vrl/web-playground
+      - run: wasm-pack test --node
+
+  master-failure:
+    name: master-failure
+    if: failure() && github.ref == 'refs/heads/master'
+    needs:
+      - changes
+      - cross-linux
+      - test-misc
+      - test-linux
+      - test-mac
+      - test-windows
+      - test-vrl
+      - check-component-features
+      - checks
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Discord notification
+      env:
+        DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+      uses: Ilshidur/action-discord@0.3.2
+      with:
+        args: "Master tests failed: <https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}>"
```

Comment:
I don't think we want this in this workflow.

I also wonder if we actually _need_ a separate workflow, as opposed to adding a single job to the existing "test VRL" workflow we already have.

I'd like to get @jszwedko's take on this, as he has more in-depth knowledge of our CI workflow structure.

Reply:
If it is a separate workflow, I think the notification make sense. I also agree that this could be folded into the existing `test.yml` workflow.